internal/clients/compliance/client.go

```diff
@@ -53,7 +54,10 @@ func (cc *ComplianceClient) request(ctx context.Context, method, url string) (*h
 	req.Header.Add("x-rh-identity", id)
 	req.Header.Add("content-type", "application/json")
 
-	return cc.client.Do(req)
+	// TODO context must be passed to the request method
+	req = req.WithContext(context.TODO())
+	doer := strc.NewTracingDoer(cc.client)
+	return doer.Do(req)
```

Comment:
Just noticed context is actually in this HTTP client, so will change this and this one will work properly.